generation/generate_gapic_bom.sh

```diff
@@ -17,8 +17,13 @@ for bom_directory in $(find . -maxdepth 3 -name 'google-*-bom' | sort --dictiona
   artifactId_line=$(grep --max-count=1 'artifactId' "${pom_file}")
   version_line=$(grep --max-count=1 'x-version-update' "${pom_file}")
 
-  if [[ "$groupId_line" == *"com.google.maps"* ]]; then
-    # The gapic bom includes cloud libraries
+  if [[ "$groupId_line" != *"com.google.cloud"*
+      && "$groupId_line" != *"com.google.analytic"*
+      && "$groupId_line" != *"com.google.area120"*
+      && "$groupId_line" != *"io.grafeas"*  ]]; then
+    # The gapic bom mainly includes cloud libraries and ones that have been included already.
+    # Let's avoid adding com.google.maps and com.google.shopping for now. We may decide to
+    # add them later. It's more difficult to remove them later without impacting users.
     continue
   fi
```

Comment:
nit: Negative booleans can sometimes increase cognitive load. How about something like this?
```suggestion
  if [[ "$groupId_line" == *"com.google.cloud"*
      || "$groupId_line" == *"com.google.analytic"*
      ||"$groupId_line" == *"com.google.area120"*
      || "$groupId_line" == *"io.grafeas"*  ]]; then
    # The gapic bom mainly includes cloud libraries and ones that have been included already.
    # Let's avoid adding com.google.maps and com.google.shopping for now. We may decide to
    # add them later. It's more difficult to remove them later without impacting users.
   bom_lines+="      <dependency>\n\
      ${groupId_line}\n\
      ${artifactId_line}\n\
      ${version_line}\n\
        <type>pom</type>\n\
        <scope>import</scope>\n\
      </dependency>\n"
  else
     continue
  fi
```

Reply:
That was also itchy part for me too. Updated.